.spelling

```diff
@@ -418,6 +420,7 @@ QSFP-DD
 quiescing
 Rados
 Rados-Gateway
+rbd
```

Comment:
This is also something which should be in backticks. For names of devices or commands and the like, they should not be in the dictionary.

```suggestion
```